motion_computation/config/parameters.yaml

```diff
@@ -18,15 +18,15 @@ prediction_confidence_drop_rate: 0.95
 
 # Boolean: If true then BSM messages will be converted to ExternalObjects. 
 #          If other object sources are enabled, they will be synchronized but no fusion will occur (objects may be duplicated)
-enable_bsm_processing: false
```

Comment:
Can we leave these configs at their default values, so false for bsm and true for psm and mobility path?